### PR TITLE
clean-up Travis CI config

### PR DESCRIPTION
**Pull request by mathias-luedtke**
RE https://discourse.ros.org/t/industrial-ci-migration-needed-for-recent-and-future-changes/9462

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 matrix:
   allow_failures:
-    - env: USE_DEB=true  ROS_DISTRO="melodic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: USE_DEB=true  ROS_DISTRO="melodic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - env: USE_DEB=true  ROS_DISTRO="melodic"   PRERELEASE=true
+    - env: ROS_DISTRO="melodic"   ROS_REPO=ros
+    - env: ROS_DISTRO="melodic"   ROS_REPO=ros-shadow-fixed
+    - env: ROS_DISTRO="melodic"   PRERELEASE=true
 install:
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```
Comment (gavanderhoorn) [inline .travis.yml]:
```suggestion
  - git clone --quiet --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config
```

**Comment by miguelprada** [comment]
Should this (temporarily) point to `colcon-rebase`, same as ros-industrial/ur_modern_driver#323?

**Comment by mathias-luedtke** [comment]
> Should this (temporarily) point to colcon-rebase, same as ros-industrial/ur_modern_driver#323?

I try to add colcon tests for the more complicated cases only (Travis is very busy..).
I just stumbled upon the outdated config ^^

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Merging this in.

:+1: @ipa-mdl.

**Comment by gavanderhoorn** [comment]
As the second commit was basically a fixup, I've squash-merged.

**Pull request merged**